thinc/tests/util.py

```diff
@@ -95,18 +96,29 @@ def check_input_converters(Y, backprop, data, n_args, kwargs_keys, type_):
     assert all(isinstance(arg, type_) for arg in Y.args)
     assert all(isinstance(arg, type_) for arg in Y.kwargs.values())
     dX = backprop(Y)
+
+    def is_supported_backend_array(arr):
+        if has_cupy:
+            import cupy
+
+            return isinstance(arr, cupy.ndarray) or isinstance(arr, numpy.ndarray)
+        else:
+            return isinstance(arr, numpy.ndarray)
```

Comment:
```suggestion
        return is_cupy_array(arr) or is_numpy_array(arr)
```

And import these functions from `thinc.util`.